src/algebra/module/linear_map.lean

```diff
@@ -951,3 +951,48 @@ lemma End.int_cast_def (z : ℤ) [add_comm_group N₁] [module R N₁] :
   (↑z : module.End R N₁) = module.to_module_End R N₁ z := rfl
 
 end module
+
+section coe
+
+variables {R S} [semiring R] [semiring S] (σ : R →+* S)
+
+/-- `coe_is_semilinear_map σ M N` is a class stating that the coercion map `↑ : M → N`
+(a.k.a. `coe`) is a `σ`-semilinear map.
```

Comment:
Perhaps something like "see the notes for `coe_is_smul_hom` for an explanation of why `coe_is_add_monoid_hom` must be provided separately"; or just repeat that note here.